question-only-right/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/question-only-right/styles.css" />
+    <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="/assets/colors.css" />
+
+    <title>Quiz App</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <h1>Quiz FinSight</h1>
```

Comment:
Would suggest to use a smaller heading tag `h2, h3 etc` or some utility class for your heading, 
while `h1` here is not wrong, since it is meant to be the title of a website, it should ideally be placed accordingly.